drivers/spi/CMakeLists.txt

```diff
@@ -34,6 +34,6 @@ zephyr_library_sources_ifdef(CONFIG_SPI_MCHP_QSPI	spi_mchp_mss_qspi.c)
 zephyr_library_sources_ifdef(CONFIG_SPI_PL022		spi_pl022.c)
 zephyr_library_sources_ifdef(CONFIG_SPI_ANDES_ATCSPI200	spi_andes_atcspi200.c)
 zephyr_library_sources_ifdef(CONFIG_NXP_S32_SPI spi_nxp_s32.c)
-
 zephyr_library_sources_ifdef(CONFIG_SPI_ASYNC spi_signal.c)
+zephyr_library_sources_ifdef(CONFIG_SPI_ITE_IT8XXX2	spi_ite_it8xxx2.c)
```

Comment:
Nitpick: This is a wrong place to add this line. IMO it should be added after line 36, and the blank line should separate this group from the more generic files (async, userspace etc.)

Reply:
Dear Filip,

Thanks for reminding, yes you're right. I'll correct this in the next upload.
Thanks again for helps.

Warmest regards,
BJ